### ISSUE TITLE
Bug in PrePostSovleMatrix

### ISSUE THREAD

**Issue by jupiters1117** [coin-or/CoinUtils#135]
https://github.com/coin-or/CoinUtils/blob/master/src/CoinPrePostsolveMatrix.cpp#L498-L501

Seems the **row status** in **PrePostSolveMatrix** is not set properly:

```
  } else if (fabs(lower - value) <= ztolzb_) {
    setRowStatus(iRow, atUpperBound); 
  } else if (fabs(upper - value) <= ztolzb_) {
    setRowStatus(iRow, atLowerBound);
```

The following makes more sense to me:

```
  } else if (fabs(lower - value) <= ztolzb_) {
    setRowStatus(iRow, atLowerBound); 
  } else if (fabs(upper - value) <= ztolzb_) {
    setRowStatus(iRow, atUpperBound);
```

**Comment by jjhforrest** [comment]
Bug makes no difference as Clp checks later - also no difference in Cbc 
(using Clp anyway).  Changed anyway.

**Comment by tkralphs** [comment]
Closed via 5ffade9
